src/kernel/chainparams.cpp

```diff
@@ -417,17 +417,15 @@ class SigNetParams : public CChainParams {
     explicit SigNetParams(const SigNetOptions& options)
     {
         std::vector<uint8_t> bin;
+        vFixedSeeds.clear();
         vSeeds.clear();
 
         if (!options.challenge) {
             bin = "512103ad5e0edad18cb1f0fc0d28a3d4f1f3e445640337489abb10404f2d1e086be430210359ef5021964fe22d6f8e05b2463c9540ce96883fe3b278760f048f5189f2e6c452ae"_hex_v_u8;
+            vFixedSeeds = std::vector<uint8_t>(std::begin(chainparams_seed_signet), std::end(chainparams_seed_signet));
```

Comment:
1) the `chainparams: add signet fixed seeds if default network` commit should go after `seeds: update fixed dns seeds` which introduces `chainparams_seed_signet`
2) Now that we're pre-clearing it at the beginning, it may be slightly simpler to do:
```suggestion
            vFixedSeeds.assign(std::begin(chainparams_seed_signet), std::end(chainparams_seed_signet));

```

Reply:
I see, what do you think about 2)?